service/src/main/kotlin/io/provenance/digitalcurrency/consortium/domain/MarkerTransfer.kt

```diff
@@ -18,17 +18,18 @@ object MarkerTransferTable : BaseRequestTable(name = "marker_transfer") {
 
 open class MarkerTransferEntityClass : BaseRequestEntityClass<MTT, MarkerTransferRecord>(MTT) {
     fun insert(fromAddress: String, denom: String, amount: String, toAddress: String, height: Long, txHash: String) =
```

Comment:
do you know what caused the dupe pk to begin with?

not a current use case but couldn't a txHash include more than one marker transfer?

Reply:
My understanding is there was 2 event stream threads and they were both processing all events? I'm not sure what would cause that or keep it constantly conflicting on every iteration.

Reply:
Sounds like that bug we had awhile back where an exception i the event stream handler started up a duplicate event stream thread. Maybe we need to reconcile that we have that fix applied here?